tvtk/pyface/picker.py

```diff
@@ -473,10 +474,10 @@ def close_picker(self):
     def text_setup(self):
         """Sets the properties of the text widget"""
         self.data.text_actor._get_text_property().font_size = 100
-        self.text_rep._get_position_coordinate().set(value=(.15, .15, 0))
```

Comment:
traits deprecation of `set`->`trait_set` (elsewhere we did this without try/except so I did it that way here)